syntax = "proto3";

package tenant_parser;

import "google/protobuf/empty.proto";

option go_package = "github.com/milkyhoop/tenant_parser/internal/delivery/pb;intentparser";
// option python_package = "app.protos";

service IntentParserService {
  rpc DoSomething (IntentParserRequest) returns (IntentParserResponse);
  rpc HealthCheck (google.protobuf.Empty) returns (google.protobuf.Empty);
}

message IntentParserRequest {
  string user_id = 1;
  string input = 2;
}

message IntentParserResponse {
  string status = 1;
  string result = 2;
}
